.gitignore

```diff
@@ -4,3 +4,6 @@
 terraform.tfstate*
 terraform.tfvars
 values.dev.yaml
+bin/
+pkg/
```

Comment:
Do we have these dirs?

Reply:
Wouldn't they be generated by running `go build` and the like?